src/Forms/Prism.Forms/Navigation/IInitialize.cs

```diff
@@ -1,7 +1,15 @@
 ﻿namespace Prism.Navigation
 {
+    /// <summary>
+    /// Synchronous initialization of navigation
+    /// </summary>
     public interface IInitialize
     {
+        /// <summary>
+        /// Synchronously initialize method which initializes the navigation.
```

Comment:
No need to add "synchronous" in the description. Invoked when the View or ViewModel is first created during the navigation process.

Reply:
done